plugins/Marketplace/vue/src/PluginDetailsModal/PluginDetailsModal.vue

```diff
@@ -85,6 +85,12 @@
           <div v-else-if="showExceededLicenseDescription" class="alert alert-warning">
             {{ translate('Marketplace_PluginLicenseExceededDescription') }}
           </div>
+          <div v-else-if="plugin.licenseStatus === 'Pending'"  class="alert alert-warning">
```

Comment:
@michalkleiner We have this 2 more statuses, should we pass it from twig to vue ?